.vscodeignore

```diff
@@ -0,0 +1,14 @@
+**/*.ts
+**/tsconfig.json
+**/*.yml
+**/*.js.map
+*ignore
+*.vsix
+package-lock.json
+img/
+node_modules/
+out/edge/
+src/
+scripts/
+startpage/

```

Comment:
We probably need to keep `startpage/index.html` since the extension defaults to accessing this page when launching a target.

Reply:
Good call. I thought this exclusion would be restricted to top level, but it was also applying to the out/startpage directory. Updated to include out/startpage but exclude startpage/

Reply:
![image](https://user-images.githubusercontent.com/42152559/122117616-ec3ec280-cddb-11eb-964f-f94e7be710aa.png)
After update